config/more_samples-glyf_colr_1.py

```diff
@@ -0,0 +1,199 @@
+"""Compile samples that are infeasible or difficult by svg compilation.
+"""
+
+import datetime
+from pathlib import Path
+from fontTools import fontBuilder
+from fontTools import ttLib
+from fontTools.colorLib import builder as colorBuilder
+from fontTools.pens.ttGlyphPen import TTGlyphPen
+from fontTools.ttLib.tables._g_l_y_f import Glyph
+import sys
+from typing import Any, Mapping, NamedTuple, Optional
+from fontTools.ttLib.tables import otTables as ot
+from nanoemoji.colors import css_colors, Color
+from fontTools.misc.transform import Transform
+
+_UPEM = 1000
+_ASCENT = 950
+_DESCENT = 250
+_FAMILY = "More COLR v1 Samples"
+_STYLE = "Regular"
+_PALETTE = {}  # <3 mutable globals
+
+
+class SampleGlyph(NamedTuple):
+    glyph_name: str
+    accessor: str
+    advance: int
+    glyph: Glyph
+    colr: Optional[Mapping[str, Any]] = None
+
+
+def _cpal(color_str):
+    color = Color.fromstring(color_str).to_ufo_color()
+    if color not in _PALETTE:
+        _PALETTE[color] = len(_PALETTE)
+    return _PALETTE[color]
+
+
+def _sample_sweep():
+    glyph_name = "sweep"
+
+    pen = TTGlyphPen(None)
+    pen.moveTo((100, 500))
+    pen.qCurveTo((500, 1000), (900, 500))
+    pen.qCurveTo((500, 0), (100, 500))
+    pen.closePath()
+
+    colr = {
+        "Format": ot.PaintFormat.PaintGlyph,
+        "Glyph": glyph_name,
+        "Paint": {
+            "Format": ot.PaintFormat.PaintSweepGradient,
+            "ColorLine": {
+                "ColorStop": [
+                    (0.0, _cpal("red")),
+                    (0.5, _cpal("yellow")),
+                    (1.0, _cpal("red")),
+                ]
+            },
+            "centerX": 400,  # TODO: looks off-center at 500
+            "centerY": 500,
```

Comment:
How to make the center point of the gradient at the center of the circle?

Reply:
@anthrotype can you sanity check? - either I'm tired and did something dumb [most likely the case] or setting center where I thought it should go didn't give the expected result

Reply:
@rsheeter I fixed it with https://github.com/googlefonts/color-fonts/pull/55/commits/18a2f72f86acba962c7bd2cdbdf7fead68620793 
Say "I consent" so @googlebot is happy

Reply:
I consent

Reply:
I think you need to mention the bot, otherwise it doesn't know you're replying to it

Reply:
nevermind, actually it was waiting on _my_ consent 🤭

Reply:
Silly lsb. It would be very helpful to have osmething that explains "how font coordinates fit together" in a slightly more friendly way than the spec. I don't do it all that often and I feel like I always forget one nuance or another.

In my dreams we get a 3b1b style video on programmatically building a font, then making it variable.